portfolio/src/main/webapp/script.js

```diff
@@ -110,6 +115,17 @@ function createListElement(text) {
   return liElement;
 }
 
+/** 
+ * Returns HTML image element from Url 
+ */
+function createImageElement(Url) {
```

Comment:
Nit picking: we use `lowerCamelCase` format for parameter names - so `url` would fine here.

Spec: https://google.github.io/styleguide/jsguide.html#naming-parameter-names